src/crate/theme/rtd/crate/navbar.html

```diff
@@ -1,3 +1,7 @@
+<!--esi
+<esi:include src="https://crate.io/navi-promo.php" />
```

Comment:
This URL currently returns an empty response body?

Reply:
The response body is not actually empty (`Content-Length: 23`) but apparently contains some newline characters.

Reply:
right, that is because the promo header is disabled in WordPress on production atm